db/migrate/20170214132230_create_ssh_keys.rb

```diff
@@ -0,0 +1,12 @@
+class CreateSshKeys < ActiveRecord::Migration
+  def change
+    create_table :ssh_keys do |t|
+      t.string :name, :limit => 255
+      t.string :key, :text, :limit => 5000
```

Comment:
Ah, good catch! Thanks.
This line should obviously read `t.text :key, :limit => 5000`